iOS Onboarding/AppDelegate.swift

```diff
@@ -5,32 +5,11 @@
 //  Created by Vlad_Rosca on 13.07.2023.
 //
 
-import UIKit
+import UIKit.UIResponder
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-
-
-  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-    // Override point for customization after application launch.
-    return true
-  }
-
-  // MARK: UISceneSession Lifecycle
-
-  func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
-    // Called when a new scene session is being created.
-    // Use this method to select a configuration to create the new scene with.
-    return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
-  }
-
-  func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
-    // Called when the user discards a scene session.
-    // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
-    // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
-  }
-
-
+  func application(_ application: UIApplication,
```

Comment:
Do you need [this API](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application)?